components/Carousel/Carousel.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Carousel extends React.PureComponent {
+  static propTypes = {
+    imageUrls: PropTypes.array,
+  };
+
+  changeImage = e => {
+    this.refs.displayImage.src = e.target.src;
+  };
+
+  render() {
+    const { imageUrls } = this.props;
+    return (
+      <React.Fragment>
+        <style jsx>{`
+          .image_container {
+            width: 50%;
+          }
+          .thumbnail_big {
+            width: 90%;
+            height: 480px;
+            margin: 20px;
+          }
+          .thumbnail_small {
+            margin: 10px;
+            width: 20%;
+            height: 66px;
+          }
+          @media (max-width: 650px) {
+            .image_container {
+              width: 100%;
+            }
+          }
+        `}</style>
+        <div className="image_container">
+          <img
+            className="thumbnail_big"
+            ref="displayImage"
```

Comment:
String refs are deprecated. Use [React.createRef](https://reactjs.org/docs/refs-and-the-dom.html#creating-refs)